core/orchestrator.go

```diff
@@ -289,7 +289,10 @@ func (orch *orchestrator) priceInfo(sender ethcommon.Address) (*big.Rat, error)
 			return nil, err
 		}
 
-		overhead = overhead.Add(overhead, new(big.Rat).Inv(txCostMultiplier))
+		if txCostMultiplier != nil && txCostMultiplier.Cmp(big.NewRat(0, 1)) > 0 {
```

Comment:
If the returned error is nil, `TxCostMultiplier()` should never return nil for the first value which means that the preceding error check should obviate the need for an additional nil check here.

I know that there are a few cases in the codebase where methods return (nil, nil), but I think those are the exceptions rather than the norms and we should probably avoid those cases going forward otherwise we'll have add 2 nil checks for every single method call instead of a single error check for every single method call.